src/cap-description.adoc

```diff
@@ -404,7 +402,7 @@ R  = Bc - 1
 .Calculation of base address correction
 [#cap_encoding_cb,options=header,cols="^1,^1,^1",width="40%",align="center"]
 |==============================================================================
-| A~c~ < R | B~c~ < R | c~b~
+| A    < R | B    < R | c
```

Comment:
This one is still cb to define the variable above for base

Reply:
Fixed